datafusion/core/src/datasource/listing/helpers.rs

```diff
@@ -671,6 +680,106 @@ mod tests {
         );
     }
 
+    fn describe_partition(partition: &Partition) -> (&str, usize, Vec<&str>) {
```

Comment:
a comment here might be nice explaining what the str/usize/Vec<&str> means for future readers